.pipelines/templates/template-deploy-e2e-db.yml

```diff
@@ -1,13 +1,15 @@
 parameters:
   workingDirectory: ""
+  rpMode: ""
 steps:
   - script: |
       cd ${{ parameters.workingDirectory }}
 
       . secrets/env
       . ./hack/e2e/run-rp-and-e2e.sh
-
+    
       export DATABASE_NAME=v4-e2e-$(git log --format=%h -n 1 HEAD)
 
       deploy_e2e_db
     displayName: "🚀 Deploy custom RP DB"
+    condition: eq(variables['RP_MODE'], 'development')
```

Comment:
Since you do `echo "##vso[task.setvariable variable=RP_MODE]$RP_MODE"` earlier in the pipeline, do you still need to declare the `rpMode` at the top here ?

Reply:
No that is a left over from switching to just using the variable instead of a parameter. I'll remove it

Reply:
removed